pyseir/rt/infer_rt.py

```diff
@@ -570,7 +586,16 @@ def infer_all(self, plot=True, shift_deaths=0):
             df_raw[timeseries_type.value] = timeseries_raw
 
             df = pd.DataFrame()
-            dates, posteriors, start_idx = self.get_posteriors(timeseries_type)
+            try:
+                dates, posteriors, start_idx = self.get_posteriors(timeseries_type)
+            except Exception as e:
+                structlog.getLogger().exception(
+                    event="Posterior Calculation Error",
+                    region=self.regional_input.display_name,
+                    error=e,
```

Comment:
If you need to explicitly pass e to `exception()` our logger is not configured correctly. https://github.com/hynek/structlog/issues/31#issuecomment-69319872

Reply:
fixed.

Reply:
Removed.